### ISSUE TITLE
Add aarch64 and ppc64le builds

### ISSUE THREAD

**Issue by matthewfeickert** [conda-forge/hepmc2-feedstock#8]
@Kwaizer what we could do to improve the feedstock is to add aarch64 and ppc64le builds by adding to the existing config file

https://github.com/conda-forge/hepmc2-feedstock/blob/9ac500d83544bf5e79dc04ee57696c3592273121/conda-forge.yml#L1-L6

the following

```yaml
build_platform:
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
  osx_arm64: osx_64
provider:
  linux_aarch64: default
  linux_ppc64le: default
test: native_and_emulated
```

This would enable more machines to be able to use the feedstock.

c.f. https://github.com/conda-forge/fastjet-feedstock/pull/13 as an example

**Comment by matthewfeickert** [comment]
@Kwaizer if you are able to knock this one out as well that would be great. :+1:

**Comment by Kwaizer** [comment]
@matthewfeickert this one seems be already merged

**Comment by matthewfeickert** [comment]
> this one seems be already merged

@Kwaizer what do you mean? Please link the PR you're talking about but you can see that there is no support here for this repo.

**Comment by matthewfeickert** [comment]
Ah you mean PR https://github.com/conda-forge/hepmc2-feedstock/pull/9. Can you check to make sure there's no differences there and then we can try to get a rerender.

**Comment by Kwaizer** [comment]
@matthewfeickert sorry, what do you mean by checking if there is no differences?

**Comment by matthewfeickert** [comment]
The builds didn't happen for the platforms as there wasn't a rerender for PR #9. I've opened Issue #10 which created PR #11 which should address this.

**Comment by Kwaizer** [comment]
You mean e. g. https://conda.anaconda.org/conda-forge/linux-aarch64/ doesn't have hepmc2 package yet, right?

**Comment by matthewfeickert** [comment]
@Kwaizer yeah correct. You can see from https://anaconda.org/conda-forge/hepmc2/files?version=2.06.11 that there is currently only Linux and macOS x86 builds

![image](https://github.com/user-attachments/assets/6e96e7b4-5cca-42cd-8a1e-0625f78f2bd6)

but PR #11 adds the other builds into the system. Just needed to rerender the feedstock after PR #9 updated the config files.

**Comment by matthewfeickert** [comment]
@Kwaizer so no further action needs to be taken by you. :) :+1:

**Comment by matthewfeickert** [comment]
Resolved by PR #11
